cmd/node-agent/README.md

```diff
@@ -0,0 +1,87 @@
+# Node Agent
```

Comment:
Rather than comment on all lines, just making note here: we should drop the stuff about configMaps now that we're going to rely on the upstream functionality